modelforge/potential/pain.py

```diff
@@ -35,11 +34,11 @@ def __init__(
         Parameters
             ----------
             embedding : torch.Module
```

Comment:
I think it would be good to improve the doc string here (i.e., give a bit more description about the expectation of the embedding), and also give an example below.

Reply:
added more informative docstrings